remote-api/src/main/java/org/alfresco/repo/webdav/LockMethod.java

```diff
@@ -1,28 +1,28 @@
-/*
```

Comment:
Didn't change anything here, don't know why it's marked, same with the exact same change below